### PR TITLE
#1131: fix closed pull request event with nil additions or deletions

### LINKED ISSUE #1131: Undefined '+' method for nil encountered in 'fill_up_event' of 'github-events.rb'
Found it here: https://github.com/objectionary/eo/actions/runs/18526159800/job/52797375968

```
E: NoMethodError: undefined method '+' for nil
   	/action/judges/github-events/github-events.rb:153:in 'fill_up_event'
   	/action/judges/github-events/github-events.rb:334:in 'block (4 levels) in <top (required)>'
```

### PR DESCRIPTION
**Pull request by Yegorov**
Closes #1131 

## Summary by CodeRabbit

* **Bug Fixes**
  * Improved robustness by handling edge cases where pull request change metrics are missing.

* **Tests**
  * Added test coverage for pull request metrics edge cases.

### CODE REVIEW

File: judges/github-events/github-events.rb [hunk 1]
```diff
@@ -150,7 +150,7 @@ def self.fill_up_event(fact, json)
         $loog.debug("New PR #{Fbe.issue(fact)} opened by #{Fbe.who(fact)}")
       when 'closed'
         fact.what = "pull-was-#{pl[:merged_at].nil? ? 'closed' : 'merged'}"
-        fact.hoc = pl[:additions] + pl[:deletions]
+        fact.hoc = (pl[:additions] || 0) + (pl[:deletions] || 0)
```
[the remark this answers is not in the document]
Comment (Yegorov) [reply judges/github-events/github-events.rb]: @coderabbitai, thanks, but this is not related to this problem and such an error for line 185 has not yet occurred.

**Comment by yegor256** [comment]
@Yegorov thanks!

**Comment by 0crat** [comment]
@yegorov Great work on your contribution! 🎉 You've earned **+22 points** (+24 base, +5 for 77 hits-of-code, -8 for missing code review). [Keep up the momentum](https://www.yegor256.com/2018/03/06/speed-vs-quality.html) - your balance is now [+661](https://zerocracy.github.io/judges-action/zerocracy-vitals.html). Don't forget to check your [Zerocracy balance](https://www.zerocracy.com/) too!
